ui/component/postEditor/view.jsx

```diff
@@ -82,7 +82,7 @@ function PostEditor(props: Props) {
       // Editing same file (previously published)
       // User can use a different file to replace the content
       if (!ready && !filePath && !fileText && streamingUrl && fileMimeType === 'text/markdown') {
-        setCurrentFileType(fileMimeType);
+        // setCurrentFileType(fileMimeType);
```

Comment:
If you drag a .md (and also .txt, I think) into the webpage, it should redirect you to Post + the post editor filled in with the file's content.

There is a legacy bug such that if the you are previously filling a publish form (file or livestream), it doesn't clear out the form when the md is dragged in.

Reply:
Seems like regular files is broken too, but also in production -- it doesn't select the file directly.  Meh, if no one complained, probably can kill the feature ...